lib/tessel/provision.js

```diff
@@ -256,7 +287,7 @@ function AlreadyAuthenticatedError() {
 util.inherits(AlreadyAuthenticatedError, Error);
 
 actions.AlreadyAuthenticatedError = AlreadyAuthenticatedError;
-
+module.exports.TESSEL_AUTH_KEY = filepath;
```

Comment:
@Student007 if the filepath variable is ever set by another part of the program, it seems like this export will store an old variable. Is that understanding correct?